cluster/gce/util.sh

```diff
@@ -869,6 +853,30 @@ function kube-push {
   echo
 }
 
+function kube-update-nodes() {
+  action=${1}
+
+  OUTPUT=${KUBE_ROOT}/_output/logs
+  mkdir -p ${OUTPUT}
+
+  echo "Updating node metadata... "
+  write-node-env
+  for (( i=0; i<${#MINION_NAMES[@]}; i++)); do
+    add-instance-metadata-from-file "${MINION_NAMES[$i]}" "kube-env=${KUBE_TEMP}/node-kube-env.yaml" &
```

Comment:
This is obviously great as a first draft; can you add a TODO for me in here to grab the `node-kube-env.yaml` and `configure-vm.sh` from the correct version? That seems like the most pressing addition needed.